third_party/workspace.bzl

```diff
@@ -46,15 +46,16 @@ def tensorboard_workspace():
   # We need to import the protobuf library under the names com_google_protobuf
   # and com_google_protobuf_cc to enable proto_library support in bazel.
   # Unfortunately there is no way to alias http_archives at the moment.
-  native.http_archive(
-      name = "com_google_protobuf",
-      urls = [
-          "http://mirror.bazel.build/github.com/google/protobuf/archive/2b7430d96aeff2bb624c8d52182ff5e4b9f7f18a.tar.gz",
-          "https://github.com/google/protobuf/archive/2b7430d96aeff2bb624c8d52182ff5e4b9f7f18a.tar.gz",
-      ],
-      sha256 = "e5d3d4e227a0f7afb8745df049bbd4d55474b158ca5aaa2a0e31099af24be1d0",
-      strip_prefix = "protobuf-2b7430d96aeff2bb624c8d52182ff5e4b9f7f18a",
-  )
+  if "com_google_protobuf" not in native.existing_rules():
```

Comment:
Why the new guard here?

Reply:
I'm honestly not sure. Something appears to be defining that and even as I grep through everything, I can't quite find out where it's coming from. It could be rules_go, which is doing strange things with their workspace definitions.

This is a simple way to punt the problem until I have time to take a closer look.

Reply:
SG! Just curious.